api/build.gradle

```diff
@@ -120,6 +120,11 @@ def swagger2JavaClient(yaml, pkg, rawOptExtras = []) {
                     'invokerPackage'   : "${SWAGGER_CODEGEN_BASE_PKG}.${pkg}",
                     'modelPackage'     : "${SWAGGER_CODEGEN_BASE_PKG}.${pkg}.model",
                     'apiPackage'       : "${SWAGGER_CODEGEN_BASE_PKG}.${pkg}.api",
+                    // java8
+                    // Option. Use Java8 classes instead of third party equivalents
```

Comment:
Could you elaborate?  I have additional questions along the lines of:
* what is being "used" here, and for what purpose?
* what does the choice here indicate?  why are the options "Java 8" and "third party" ?
* why is it necessary to specify this now?  why won't Java 11 work here? does Java 11 lack Base64? why do we need Base64?

Reply:
Seemed like a good idea to me based on that documentation, but I did a diff and could not find any benefits, so I reverted.